src/components/ContentRenderer.tsx

```diff
@@ -13,24 +15,73 @@ type Props = {
 
 const ContentRenderer: React.FC<Props> = ({ contentBlocks }) => {
   return (
-    <FullWidthContainer className="relative p-3">
+    <FullWidthContainer className="relative">
```

Comment:
There's definitely more cleanup to be done (why is this `position: relative`?) but that's a wider issue that's better addressed by future work.